### PR TITLE
[TM-6077] Re-Integrate Publishable Positions Edit

### PR DESCRIPTION
**Pull request by jlam-mpc**
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/1004)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/479)

[TM-6077](https://metaphase.atlassian.net/browse/TM-6077)

[TM-6077]: https://metaphase.atlassian.net/browse/TM-6077?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/PublishablePositionCard/PublishablePositionCard.jsx [hunk 1]
```diff
@@ -107,10 +107,17 @@ const PublishablePositionCard = ({
 
   const onSubmitForm = () => {
     const editData = {
+      psCD: data?.psCD,
+      aptSeqNum: data?.aptSeqNum,
+      created: data?.ORIGpposcreatetmsmpdt.replace(/T/g, " "),
```
Comment (elizabeth-jimenez) [inline src/Components/PublishablePositionCard/PublishablePositionCard.jsx:112]: 🫡

Comment (elizabeth-jimenez) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx:112]: no wait, there is no ORIG of this one

File: src/Components/PublishablePositionCard/PublishablePositionCard.jsx [hunk 2]
```diff
@@ -107,10 +107,17 @@ const PublishablePositionCard = ({
...
+      createdUserID: data?.pposcreateuserid,
+      positionDetailsLastUpdated: data?.ORIGpositionDetailsLastUpdated.replace(/T/g, " "),
```
Comment (elizabeth-jimenez) [inline src/Components/PublishablePositionCard/PublishablePositionCard.jsx:114]: 🫡

File: src/Components/PublishablePositionCard/PublishablePositionCard.jsx [hunk 3]
```diff
@@ -107,10 +107,17 @@ const PublishablePositionCard = ({
...
+      posAuditExclusionInd: data?.posAuditExclusionInd,
+
       posSeqNum: data?.posSeqNum,
       positionDetails: textArea,
       lastUpdatedUserID: data?.positionLastUpdatedUserID,
-      lastUpdated: data?.ORIGpositionLastUpdated,
+      lastUpdated: data?.ORIGpositionLastUpdated.replace(/T/g, " "),
```
Comment (elizabeth-jimenez) [inline src/Components/PublishablePositionCard/PublishablePositionCard.jsx:120]: 🫡

File: src/Components/PublishablePositionCard/PublishablePositionCard.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jlam-mpc) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx]: The create request looks like this
```
Data: {
  "pposposseqnum": 774,
  "ppospubscd": "P",
  "pposaptsequencenum": null,
  "pposcreatetmsmpdt": "2003-07-01 22:16:43",
  "pposcreateuserid": 1,
  "pposcapsulemodifydt": "2020-10-16 11:40:57",
  "pposauditexclusionind": "N",
  "pposcapsuledescrtxt": "Scott testing this capsule",
  "pposlastupdttmsmpdt": "2020-10-16 11:40:57",
  "pposlastupdtuserid": 53875
}
```

`pposcreatetmsmpdt` is different from `poscreatedate`

[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx]: ur too quick jeanie, i edited it but i was too slow for u

[the remark this answers is not in the document]
Comment (jlam-mpc) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx]: Oh but I want to grab the unformatted one to remove the T for editing so I'm using ORIG

[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx]: sorry, i hadnt seen ur updated API so i wasnt seeing it in the FE payload! disregard!

Comment (elizabeth-jimenez) [reply src/Components/PublishablePositionCard/PublishablePositionCard.jsx]: disregard! there is now

**Pull request merged**